buildSrc/src/main/java/Dependencies.kt

```diff
@@ -82,6 +83,7 @@ object Dependencies {
     const val testing_coroutines = "org.jetbrains.kotlinx:kotlinx-coroutines-test:${Versions.coroutines}"
     const val testing_maven_ant_tasks = "org.apache.maven:maven-ant-tasks:${Versions.maven_ant_tasks}"
     const val testing_leakcanary = "com.squareup.leakcanary:leakcanary-android-instrumentation:${Versions.leakcanary}"
+    const val testing_json = "org.json:json:${Versions.json}"
```

Comment:
Instead of importing this you may want to run the test with the `AndroidJUnit4` runner, which will make this and other APIs available.

Reply:
Ah, that's what I've missed.  Thanks I'll update.